scss/_mixins.scss

```diff
@@ -8,7 +8,18 @@
 /// @param {Integer} $iconset-version [1] - At present only 1 version of the icon set is available.
 
 @mixin oIconsContentIcon($icon-name, $color: null, $size: 128, $include-base-styles: true, $high-contrast-fallback: true, $iconset-version: 1) {
+	// Error if the global $system-code variable is not set.
+	// This is required for image service requests.
+	@if(global-variable-exists('system-code') == false or type-of($system-code) != 'string') {
+		@error 'A global "$system-code" Sass variable must be set to a valid Bizops system code (\e]8;;https://biz-ops.in.ft.com/list/Systems\e\\https://biz-ops.in.ft.com/list/Systems\e]8;;\e\\).';
```

Comment:
We should check we have this written in the Origami specification somewhere and link to it from within this error message